packages/slider/test/slider.test.ts

```diff
@@ -1378,100 +1378,6 @@ describe('Slider', () => {
         await elementUpdated(el);
         expect(el.values).to.deep.equal({ a: 10, b: 10, c: 10 });
     });
-    it('warns in Dev Mode when `min="previous"` is leveraged on first handle', async () => {
```

Comment:
Did not remove these, just moved them below to be all inside the same `describe('dev mode')` block